packages/composite-checkout-wpcom/src/types/wpcom-store-state.ts

```diff
@@ -15,15 +15,20 @@ export function isValid( arg: ManagedValue ): boolean {
 	return arg.errors?.length <= 0;
 }
 
-function initialManagedValue( value: string ): ManagedValue {
+function getInitialManagedValue( initialProperties?: {
```

Comment:
This is probably bikeshedding, but I'm of two minds on using the `get` prefix for a pure function like this. It probably comes from working on the backend, but I instinctively expect the act of getting to touch a database at some point. :)

Reply:
This is totally a subjective discussion so I'll leave it to you to decide. 

My opinion is that functions should always start with a verb firstly to make it clear that the object is a function and not just a variable (I might write `const myValue = initialManagedValue` accidentally and it looks right - much less relevant in a language like Ruby where functions can be called without parenthesis) and also to help [clarify ambiguity around the function's purpose](https://github.com/Automattic/phpcs-neutron-standard/blob/master/GUIDELINES.md#function-naming) (although in this case it's pretty clear what the function does).